docs/user-guide/exploring-stories.md

```diff
@@ -78,6 +79,14 @@ Inside the **Social** section, social links like Facebook, Twitter and LinkedIn
 
 <img src="../img/exploring-stories/share-social.jpg" class="ms-docimage" width="400px"/>
 
-Through the **Advanced Options** it is possible to include an Home button to allow the possibility to brings the user to the MapStore Home Page: that button will be automatically included in view mode inside the story top bar just beside the navigation bar.
+Through the **Advanced Options** it is possible include to the URL:
 
-<img src="../img/exploring-stories/share-page.jpg" class="ms-docimage"/>
\ No newline at end of file
+<img src="../img/exploring-stories/share_options.jpg" class="ms-docimage" width="400px"/>
+
+* The *Home button* to allow the possibility to bring the user to the MapStore Home Page: that button will be automatically included in view mode inside the story top bar just beside the navigation bar.
+
+<img src="../img/exploring-stories/share-page.jpg" class="ms-docimage"/>
+
+* The *scroll position* to share the story section that the admin is viewing at that time on the screen.
```

Comment:
* The *scroll position* provides a share URL of the current section visualized by the user.